scripts/lint-blog.js

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+
+import fs from "node:fs";
+
+const blogPostDir = new URL("../website/blog/", import.meta.url);
+
+const files = fs.readdirSync(blogPostDir);
```

Comment:
Why not use async version? Just curious.

Reply:
There is no reason..